### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by ssaviak**
- [DEMO LINK](https://ssaviak.github.io/layout_catalog/)
- [TEST REPORT LINK](https://ssaviak.github.io/layout_catalog/report/html_report/)

- [x] All component follow BEM and use SCSS
- [x] repaeted sizes and special colors are put to variables
- [x]  Grid is used for the columns
- [x] cards are shown in 1, 2, 3 or 4 columns based on screen resolution
- [x] All changes on :hover are smooth
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,434 @@
...
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item" data-qa="nav-hover">
+            <a href="#Laptops&Computers"
+            class="nav__link"
+            >
+              Laptops & Computers
+            </a>
```
Comment (dinaminahi) [inline src/index.html:46]:
```suggestion
            <a 
               href="#Laptops&Computers"
               class="nav__link"
            >
              Laptops & Computers
            </a>
```

image

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
```
Comment (SerhiiKirik) [inline src/styles/blocks/card.scss:2]: would be better to move it to the `*`
```suggestion
```

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  width: 200px;
+  height: 408px;
+  padding:  32px 16px 16px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```
Comment (SerhiiKirik) [inline src/styles/blocks/card.scss:7]: move it to the `html` & remove in all cases
```suggestion
```

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    margin-inline: 3px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+   display: flex;
+   flex-direction: column;
+   justify-content: space-between;
+  }
```
Comment (SerhiiKirik) [inline src/styles/blocks/card.scss:25]:
```suggestion
  &__info {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
  }
```

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    align-items: center;
+    position: relative;
+    margin-top: 16px;
+    width: 100%;
+    height: 40px;
+
+    background-color: $blue-color;
+    border-radius: 5px;
+    border: none;
+    color:$white-color;
```
Comment (SerhiiKirik) [inline src/styles/blocks/card.scss:99]:
```suggestion
    color: $white-color;
```

File: src/styles/blocks/stars.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  &__star:first-child {
+    margin-left: 0;
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} > :nth-of-type(-n + 4) {
+    background-image: url(../images/star-active.svg);
+    }
+  }
+
```
Comment (SerhiiKirik) [inline src/styles/blocks/stars.scss:21]:
```suggestion
```

File: src/index.html [hunk 7]
```diff
@@ -5,8 +5,432 @@
...
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class = "catalog">
+      <div class="card" data-qa="card">
+        <img
```
Comment (SerhiiKirik) [inline src/index.html:67]:
```suggestion
    </header>
    
    <main class = "catalog">
      <div class="card" data-qa="card">
        <img
```

File: src/index.html [hunk 8]
```diff
@@ -5,8 +5,432 @@
...
+          <button class="card__button">
+            <a class="card__link" data-qa="card-hover">
+             BUY
+            </a>
+          </button>
+        </div>
+      </div>
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
```
Comment (SerhiiKirik) [inline src/index.html:114]:
```suggestion
          </button>
        </div>
      </div>
      
      <div class="card" data-qa="card">
        <img
          class="card__image"
```

File: src/index.html [hunk 9]
```diff
@@ -5,8 +5,432 @@
...
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="index.html">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
```
Comment (SerhiiKirik) [inline src/index.html:30]:
```suggestion
        <img src="images/logo.png" alt="logo">
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#Apple" class="nav__link is-active">Apple</a>
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Let's redeploy your page 🧑‍💻 
image

**Review by YegorKorenkov — APPROVED** [review]
Good job!

**Pull request closed without merging**